### ISSUE TITLE
Sequence length warmup?

### ISSUE THREAD

**Issue by cbirchsy** [kuleshov-group/caduceus#57]
Hello,

Just wondering if any sequence length warm-up  was used in the pre-training, like in HyenaDNA? 

In the loss plots in the paper it looks like the loss in the 131k plots begins close to where it ends in 32k plot above it etc.

Thanks.

**Comment by yair-schiff** [comment]
Thanks for the question. We actually did **not** use sequence length warmup. Early on in the project, we found that it wasn't necessary.
